src/languages/en.ts

```diff
@@ -2340,6 +2340,8 @@ const translations = {
             validationAmounts: 'The validation amounts you entered are incorrect. Please double check your bank statement and try again.',
             fullName: 'Please enter a valid full name',
             ownershipPercentage: 'Please enter a valid percentage number',
+            deletePaymentBankAccount:
```

Comment:
Given that we're getting the copy from the backend, let's double check with the team https://github.com/Expensify/App/issues/62778#issuecomment-2939852066